hr_contract_default_trial_length/static/description/index.html

```diff
@@ -0,0 +1,453 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I don't think this file is needed?

Reply:
Yes yes, already fixed it, did something wrong cherry-picking :sweat_smile: